bundles/org.eclipse.swt/Eclipse SWT/gtk/org/eclipse/swt/widgets/Shell.java

```diff
@@ -2030,6 +2031,7 @@ long notifyState (long object, long arg0) {
 public void open () {
 	checkWidget ();
 	setVisible (true);
+	bringToTop (true);
```

Comment:
Please move the explanation for `true` from commit message to code comment here.

Reply:
Done.